sdk/tracing/azure-core-tracing-opencensus/src/main/java/com/azure/core/tracing/opencensus/OpenCensusTracer.java

```diff
@@ -26,15 +26,49 @@
 
 /**
  * Basic tracing implementation class for use with REST and AMQP Service Clients to create {@link Span} and in-process
- * context propagation.
+ * context propagation. Singleton OpenCensus tracer capable of starting and exporting spans.
  *
  * <p>
  * This helper class supports W3C distributed tracing protocol and injects SpanContext into the outgoing HTTP
  * and AMQP requests.
  */
 public class OpenCensusTracer implements com.azure.core.util.tracing.Tracer {
-    // Singleton OpenCensus tracer capable of starting and exporting spans.
     private static final Tracer TRACER = Tracing.getTracer();
+    /**
+     * Key for {@link Context} which indicates that the context contains OpenCensus span data. This span will be used
+     * as the parent span for all spans the SDK creates.
+     * <p>
+     * If no span data is listed when the SDK creates its first span, this span key will be used as the parent span.
+     */
+    public static final String OPENCENSUS_SPAN_KEY = "opencensus-span";
+
+    /**
+     * Key for {@link Context} which indicates that the context contains the name for the OpenCensus spans that are
+     * created.
+     * <p>
+     * If no span name is listed when the span is created it will default to using the calling method's name.
+     */
+    public static final String OPENCENSUS_SPAN_NAME_KEY = "opencensus-span-name";
+
+    /**
+     * Key for {@link Context} which indicates that the context contains an entity path.
+     */
+    public static final String ENTITY_PATH = "entity-path";
```

Comment:
Are these constants used? I don't see you referencing them since they were copy and pasted in the associated packages.

Reply:
I think we should have them in the tracing package for users to use them as we look up data for these specific keys.

Reply:
For now let's not add these in. We can follow-up with them but I don't want to rush to do it for GA and have regrets

Reply:
I'm OK with this for open census. Lets revisit when the open telemetry module comes online.